action/store/postgres.py

```diff
@@ -86,8 +90,10 @@ def read(self) -> dict:
             table["fields"] = fields
             table_lookup[table_key] = table
 
+        metacopy = self.meta.copy()
```

Comment:
Just `meta` is fine, since they're scoped. Otherwise, I would prefer something like `meta_output` or something that suggests why it's being copied (but just `meta` is sufficient)